### PR TITLE
Suggested Reading added in default Layout

### PR DESCRIPTION
**Pull request by iArchitSharma**
**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit cc3df09531563c68f16770dc092694d14c19b290 at: https://6600672c3840ca0a91d9af7b--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@iArchitSharma Let's discuss this on websites call. Add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#), if you would :)

**Comment by l5io** [comment]
🚀 Preview for commit 1accdb872b379fa0f267c691e96bb643c6efbc68 at: https://660148c1ab32b2a71f8e16b8--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 959b927acb76c5b508e3ed52568c0706f5037905 at: https://660161b10aceb60608c72dd8--meshery-docs-preview.netlify.app

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -61,7 +61,11 @@ <h5>
             </div>
             {% endfor %}
           </div>
-          {% endif %} {% include_cached feedback.html %}
+          {% endif %}  
+          {% if page.suggested-reading != false and page.title and page.type and page.category and page.url %}
```
Comment (leecalcote) [inline docs/_layouts/default.html:65]: Nice work on including suggested reading by default on all pages unless their frontmatter explicitly denies its use. Good call, @iArchitSharma

**Review by leecalcote — APPROVED** [review]
A solid approach. I believe that there are many other pages that have Suggested Reading sections that need to be removed. Please confirm.

**Comment by l5io** [comment]
🚀 Preview for commit ff8c556209ef1357eaa51bf1e5ae5cfc2a9fb3f6 at: https://6602a9a80cdeb1145a2668ae--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4c3a003de31d3913001a6db6c161bcc04d5b8cc0 at: https://6602ab92493511166ef0cda5--meshery-docs-preview.netlify.app

**Comment by iArchitSharma** [comment]
@leecalcote yes with my recent commit I've removed it for all the page which are using default layout

**Comment by l5io** [comment]
🚀 Preview for commit 3181a3cf7f2bca5e070797b721b4b192bd1754bb at: https://6602cf8aee31723518683eee--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b759517994ffda2b3f67b8d15011199137e905b9 at: https://66044c08b4e9902114c37a72--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**